ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -69,7 +70,7 @@ class FaultInjector
   void jointStateCb(const sensor_msgs::JointStateConstPtr& msg);
 
   //Setting the correct values for system faults and arm faults messages
-  void setSytemFaultsMessage(ow_faults::SystemFaults& msg, int value);
+  void setSytemFaultsMessage(ow_faults::SystemFaults& msg, std::bitset<9> systemFaultsBitmask);
```

Comment:
You could have kept the function definition as is and then pass the value systemFaultBitmask.to_ullong() as a parameter. No strong feel either way!

Reply:
thanks for the great comments Ussama!